_posts/2023-01-01-seraphis-jamtis-developer-opportunities.md

```diff
@@ -0,0 +1,57 @@
+
+The current plan is to fully replace the current component with this role called *wallet2*. This in turn will make many adjustments in the rest of the Monero core codebase necessary, and of course reworking all wallet apps to use the new component.
+
+If everything works out Seraphis and Jamtis may go into service with a hardfork that some still very rough estimates see happening in about 2 years.
+
+### Financial Matters
+
+There is no company behind Monero, it has no employees. Devs seeking more or less conventional employment best look elsewhere. No venture capital companies engage in funding any work either. Monero runs on contributions from people that basically work "for free", and on donations from the broader community to finance larger tasks.
+
+There is the *CCS*, the *Community Crowdfunding System*. Its home is [here](https://ccs.getmonero.org/) on the Monero website.
+
+Basically anybody can write a proposal for it, and if it gets positive feedback it moves to the funding stage where of course success depends on whether donators will find the proposed work and goal worthwhile.
+
+Realistically however it's pretty hard to get a proposal accepted if you are all new to Monero, nobody in the community knows you yet and you have never contributed anything so far. Much is about trust and demonstrated abilities, as it typically is the case in open source projects.
+
+The CCS has a long history of successfully funded and executed tasks.
+
+A second option for funding Monero related work is the [MAGIC Monero Fund](https://magicgrants.org/funds/monero). Its nature is quite different from the CCS as it's a USA-registered 501(c)(3) charity. This enables tax-deductible (for citizens of the USA) donations in a variety of methods, and the option of USD-denominated payouts. There is also an additional requirement that fund recipients must verify their identity. The CCS allows its fund recipients to remain anonymous, yet only handles XMR.
+
+### Summary
+
+There will be a lot of interesting work to do for devs that want to join the Monero project, for a long time, even after the hardfork to Seraphis and Jamtis in a couple of years, basically starting anytime. It may all look a bit unusual as a "place" to work at, and "get rich quick" isn't, but on the other hand think about it: Where else can you make a difference for a multi-billion-dollar venture that may become important in the future and make a difference, whoever you are, from wherever you are, regardless of education, only based on your knowledge and your willingness to make useful contributions?
```

Comment:
This seems to repeat the points said elsewhere, but there is already a summary-tldr with similar content. Maybe better remove or change title and scope?

Reply:
I changed the title to *Conclusion*, and took out the repeated mention of Seraphis and Jamtis to shorten a bit.